### PR TITLE
Features/authentication

### PR DESCRIPTION
**Pull request by samyakshah3008**
Here is the Authentication for TripleS Store 

Features: 
- User can login with db email id and password as of now
- User can create new account

### CODE REVIEW

File: src/components/NavBar.jsx [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
             <input type="text" className="nav-input" placeholder="Search" />
             <span className="fas fa-search search-btn"></span>
           </div>
           <div className="nav-btn-container">
             <div className="log-in-container">
               <i className="far fa-user-circle"></i>
-              <span className="log-in">Log in</span>
+              <Link to="/login">
+                <span className="log-in">
+                  {" "}
```
Comment (anuj-kumary) [inline src/components/NavBar.jsx:23]: Remove this curly braces

**Review by anuj-kumary — APPROVED** [review]
Clean and well structured code. Well placed comments wherever required. Good to merge!

**Review by OmKakatkar — APPROVED** [review]
Looks good

I liked:
* The folder structure with separate folders for components, context and pages
* The variable and function names, seem to be meaningful

Suggestions:
* You can try using barrel imports to combine all the exports form Pages and Components. This way, you will have a single file to look for while importing them in the future

**Review by Rohanmond — APPROVED** [review]
code is looking good to merge

**Review by mdarif6 — APPROVED** [review]
- looking good folder structure.
- coding part is okay, ready to merge.

**Pull request merged**